Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3619 @@
+{
```

Comment:
<p>if I am the client am I now wondering why I was asked to download data that wont be used in the model no matter what? im paying per data point. can you make downloading that data optional? - this isnt necessary but definitely might come up if they are aware we can pull data by dates.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Added a note for it in the markdown</p>